nova/conf/vmware.py

```diff
@@ -338,6 +338,18 @@
 Possible values:
 
 * Any string or empty to keep VMware default
+"""),
+    cfg.StrOpt('bigvm_provider_vm_group',
+               help="""
+DRS VM group to put any spawned big VM as identified by `utils.is_big_vm()`
```

Comment:
Maybe we should not specify any concrete function names like utils.is_big_vm(), since the name can be changed in future and lead to confusion later in time.